package-lock.json

```diff
@@ -1,4659 +1,8 @@
 {
   "name": "amazon-chime-sdk-js",
   "version": "2.12.0",
-  "lockfileVersion": 2,
```

Comment:
Hmm could you not commit this change? It reverts the lockfile back to v1. Alternatively, please use Node 16 :)